### PR TITLE
Add common proxy headers to nginx location stanzas

### PR DESCRIPTION
**Pull request by jdbaldry**

### CODE REVIEW

File: prometheus-ksonnet/lib/prometheus.libsonnet [hunk 1]
```diff
@@ -65,7 +65,7 @@
   local pvc = $.core.v1.persistentVolumeClaim,
 
   prometheus_pvc::
-    if ! $._config.stateful
+    if !$._config.stateful
```
Comment (tomwilkie) [inline prometheus-ksonnet/lib/prometheus.libsonnet]: I believe this is a syntax error introduce by an old version of jsonnet fmt.  See https://github.com/google/jsonnet/issues/572

There is a workaround using brackets on that issue.

**Comment by tomwilkie** [comment]
Thanks for the PR @jdbaldry! One issue otherwise LGTM.

**Comment by jdbaldry** [comment]
No problem @tomwilkie. Thanks for the tip. I probably should change my workflow from running a `make fmt` at the end without retesting. Should be all good now.

**Comment by tomwilkie** [comment]
Thanks Jack!  Sorry for being slow.

**Pull request merged**